lib/client/index.js

```diff
@@ -387,13 +387,13 @@ client.init = function init(serverSettings, plugins) {
   function sgvToColor(sgv) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2447185760/issues/source?bid=2452379&fileBranchId=3246917#l387)